Sources/Amplitude/Configuration.swift

```diff
@@ -46,7 +46,7 @@ public class Configuration {
         flushMaxRetries: Int = Constants.Configuration.FLUSH_MAX_RETRIES,
         useBatch: Bool = false,
         serverZone: ServerZone = ServerZone.US,
-        serverUrl: String = Constants.DEFAULT_API_HOST,
+        serverUrl: String = "",
```

Comment:
Thanks for the PR! Minor, could we change this to `serverUrl: String? = nil`?

Reply:
@liuyang1520 Sure, done =)